app/uk/gov/hmrc/gform/sharedmodel/SmartString.scala

```diff
@@ -122,11 +126,120 @@ object SmartStringTemplateReader {
       ss              <- buildSmartStringFromLocalisedStringAndInterpolations(localisedString, interpolations)
     } yield ss
 
+  private def readFromTemplateArray(array: JsArray): JsResult[SmartString] = {
+    val jsObjects = array.value.collect { case jsObj: JsObject => jsObj }
+    val abc = jsObjects.map { jsObj =>
```

Comment:
better name

Reply:
I just wanted it to be in line with other `readFromTemplate...` methods here. It is play-json String, Object or Array
```  val templateReads: Reads[SmartString] = new Reads[SmartString] {
    override def reads(json: JsValue): JsResult[SmartString] = json match {
      case JsString(value) => readFromTemplateString(value)
      case obj: JsObject   => readFromTemplateObject(obj.fields.toMap)
      case arr: JsArray =>
        readFromTemplateArray(arr)
      case _ => JsError(s"Expected a String or an Object while reading a SmartString. Got $json")
    }
```

Reply:
I was referring to `abc`

Reply:
changed it....